components/scream/tests/coupled/shoc_cld_p3_rrtmgp/shoc_cld_p3_rrtmgp_output.yaml

```diff
@@ -0,0 +1,62 @@
+%YAML 1.1
+---
+Casename: shoc_cld_p3_rrtmgp_output_np${MPI_RANKS}
```

Comment:
Side Q: Having to create multiple input files just for the sake of differentiate the output files depending on the number of ranks is getting annoying. What do you think about making the output class incorporate npN in the file name? We could control whether we want that or not with a simple input file parameter, just like `Timestamp in Filename`...

If the idea sounds good, I'm ok either merging this and doing it in another PR as well as doing it in this PR.